tests/Integration/ConcurrencyTest.php

```diff
@@ -53,8 +53,10 @@ protected static function createEventStore(): EventStore
     private static function connection(): Connection
     {
         if (self::$connection === null) {
+            require __DIR__ . '/../../vendor/symfony/polyfill-php85/bootstrap.php';
             require __DIR__ . '/../../vendor/laravel/framework/src/Illuminate/Support/helpers.php';
             require __DIR__ . '/../../vendor/laravel/framework/src/Illuminate/Collections/helpers.php';
+            require __DIR__ . '/../../vendor/laravel/framework/src/Illuminate/Collections/functions.php';
```

Comment:
Those changes are unrelated, but I needed these lines to make `composer test:concurrency` work

Reply:
I think we need to require this? 🤔

Reply:
This should fix it. With lowest deps that file is not there and at least for me nothing failed without it 🤷

```suggestion
            if (file_exists(__DIR__ . '/../../vendor/symfony/polyfill-php85/bootstrap.php')) {
                require __DIR__ . '/../../vendor/symfony/polyfill-php85/bootstrap.php';
            }
```